tests/test_server.py

```diff
@@ -1235,33 +1233,9 @@ async def test_send_deploy_done_error_handling(server, client, environment, agen
 
     rvid_r1_v1 = ResourceVersionIdStr(f"std::testing::NullResource[agent1,name=file1],v={model_version}")
 
-    # Resource doesn't exist
-    with pytest.raises(ValueError) as exec_info:
-        await update_manager.send_deploy_done(
-            attribute_hash="",
-            result=executor.DeployReport(
-                rvid=rvid_r1_v1,
-                action_id=uuid.uuid4(),
-                resource_state=const.HandlerResourceState.deployed,
-                messages=[],
-                changes={},
-                change=const.Change.nochange,
-            ),
-            state=state.ResourceState(
-                compliance=state.Compliance.COMPLIANT,
-                last_deploy_result=state.DeployResult.DEPLOYED,
-                blocked=state.Blocked.NOT_BLOCKED,
-                last_deployed=datetime.now().astimezone(),
-            ),
-            started=datetime.now().astimezone(),
-            finished=datetime.now().astimezone(),
-        )
-    assert "The resource with the given id does not exist in the given environment" in str(exec_info.value)
-
```

Comment:
@wouterdb Do we no longer care about this case? Now that we don't fetch the resource, we don't assert that it exists

Reply:
It should always exist in the RPS table, if it is deployed by the scheduler